frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss

```diff
@@ -16,6 +16,7 @@
   .form-row,
   .content-item,
   .val-row,
+  .val-item,
```

Comment:
Oh sweet, does this cover the year alignment?

Reply:
Yes, this should fix the problem on the building form as well.